docs/dashboards/custom-component.md

```diff
@@ -79,7 +79,7 @@ Dashboards.board({
     }]
 });
 ```
-[The live example can be found here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboards/components/custom-component).
+[The live example can be found here](https://www.highcharts.com/samples/embed/dashboards/components/custom-component).
```

Comment:
Comment: This demo looks not right.

Reply:
Why?

Reply:
![Screenshot 2023-07-07 at 09 33 41](https://github.com/highcharts/highcharts/assets/40056287/a7ff2dad-0eda-4bd9-a888-751fd9e76333)

Reply:
It looks fine on my computer 🤔 😓